pkg/virt-launcher/virtwrap/device/hostdevice/gpu/hostdev_test.go

```diff
@@ -93,7 +93,7 @@ var _ = Describe("GPU HostDevice", func() {
 			RamFB:   "on",
 		}
 
-		Expect(hostDevices, err).To(Equal([]api.HostDevice{expectHostDevice0, expectHostDevice1}))
+		Expect(hostDevices).To(Equal([]api.HostDevice{expectHostDevice0, expectHostDevice1}))
```

Comment:
ditto: you can remove lines 74/75 on this file and inline the `hostDevices` call.